README.md

```diff
@@ -1,18 +1,20 @@
-The source code for the Pomona Shuttle Share application. Created by Jesse Pollak, maintained by ASPC.
```

Comment:
Eek. I never meant to do that; I think that slipped in as a result of copy pasting the README from adec372 (as it didn't contain that before) and then copying the rest from the `local_dev` branch. Will revert.